cmd/krel/cmd/ff.go

```diff
@@ -59,10 +60,10 @@ func init() {
 	rootCmd.AddCommand(ffCmd)
 }
 
-func runFf(opts *ffOptions) error {
+func runFf(opts *ffOptions, rootOpts *rootOptions) error {
```

Comment:
We have to pass the root opts here to ensure that the global modification in tests does not break other tests.